src/commands/chat/push/device/add.js

```diff
@@ -8,7 +8,7 @@ class DeviceAdd extends Command {
 		const { flags } = this.parse(DeviceAdd);
 
 		try {
-			if (!flags.user_id || !flags.provider || !flags.provider) {
+			if (!flags.user_id || !flags.provider || !flags.device_id) {
```

Comment:
@nparsons08 didn't want to touch into logic but isn't it better reverse it to be positive?